option optimize_for = LITE_RUNTIME;

message KingFightBuyCntAnswer
{
 required uint32 result = 1;
 required uint32 fight_cnt = 2;
 required uint32 total_buy_cnt = 3;
}
